rebar.config

```diff
@@ -22,8 +22,7 @@
   {smtp_rfc822_parse, return_error, 2}
  ]}.
 
-{deps, [{ranch, "^1.6.2 || ^2"},
-        {hut, "1.3.0"}]}.
+{deps, [{ranch, ">= 1.6.2 or ~> 2.0"}, {hut, "~> 1.3"}]}.
```

Comment:
`>= 1.6.2` would match 3.0 when it comes out.

Since we want to skip over 1.7, I think we need:
```
~> 1.6.2 or ~> 2.0
```
which is equivalent to:
```
(>= 1.6.2 and < 1.7.0) or (>= 2.0.0 < 3.0.0)

Reply:
Good catch!